docs/UserGuide.md

```diff
@@ -415,9 +424,13 @@ _Details coming soon ..._
-| s/     | Session timing of lessons              | A String of input in the format of`DDD HH:mm` <br>where DDD is three characters making up the day, HH are two numbers making up the hour and <br> mm are two numbers making up the minute. | s/tue 08:30    |
+| Prefix | Definition                             | Usage                                                                                                                                                                                 | Example              |
+|--------|----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| a/     | Attendance                             | A string of input in the format of `YYYY-MM-DD` <br>where YYYY are four numbers making up the year, MM are two numbers making up the month and DD are two numbers making up the day.  | a/2022-12-08         |
+| g/     | Subject and its grade                  | A string of characters                                                                                                                                                                | g/Math: A+           |
+| h/     | Homework                               | A string of characters                                                                                                                                                                | hw/Science Worksheet |
+| lp/    | Lesson plan                            | A string of characters                                                                                                                                                                | lp/Diversity         |
+| n/     | Name                                   | A string of alphanumeric characters. Whitespaces are allowed. Required                                                                                                                | n/Alex Ho            |
+| p/     | Phone number                           | Numbers only. Required. Whitespaces not allowed and at least 3 digits.                                                                                                                | p/97402341           |
+| s/     | Session timing of lessons              | A string of input in the format of`DDD HH:mm` <br>where DDD is three characters making up the day, HH are two numbers making up the hour and mm are two numbers making up the minute. | s/tue 08:30          |
+| t/     | Tag that is associated to the student. | A string of characters. Optional.                                                                                                                                                     | t/Star student       |
```

Comment:
imo the Usage column sounds too tech-y and not v friendly to users? biggest offender is the word "string" since it seems like a term only people with programming experience would know, "input" and "whitespaces" are a bit iffy but still ok

Reply:
Okies, will change the mentioned terms to a more user-friendly ones.